virttest/qemu_vm.py

```diff
@@ -1623,6 +1642,18 @@ def sort_key(dev):
                                     numa_mem, numa_cpus, numa_nodeid)
             devices.insert(StrDev('numa', cmdline=cmdline))
 
+        for numa_cpu in params.objects("guest_numa_cpus"):
+            numa_cpu_params = params.object_params(numa_cpu)
+            numa_cpu_nodeid = numa_cpu_params.get("numa_cpu_nodeid")
+            numa_cpu_socketid = numa_cpu_params.get("numa_cpu_socketid")
+            numa_cpu_dieid = numa_cpu_params.get("numa_cpu_dieid")
+            numa_cpu_coreid = numa_cpu_params.get("numa_cpu_coreid")
+            numa_cpu_threadid = numa_cpu_params.get("numa_cpu_threadid")
+            cmdline = add_numa_cpu(devices, numa_cpu_nodeid, numa_cpu_socketid,
+                                   numa_cpu_dieid, numa_cpu_coreid,
+                                   numa_cpu_threadid)
+            devices.insert(StrDev('numa', cmdline=cmdline))
```

Comment:
I think better to assign a different ID to each node to distinguish them. Not only here, but also `# Add numa nodes` above.

Reply:
> I think better to assign a different ID to each node to distinguish them. Not only here, but also `# Add numa nodes` above.
Yes, i should distinguish numa cpu and numa node, but assign different ID for each node is not very reasonable i think.
It is a type as my understanding:
https://github.com/avocado-framework/avocado-vt/blob/db232b71a3feaaab7737c8b02cf450acf4301b78/virttest/qemu_devices/qdevices.py#L348